pages/common/traceroute.md

```diff
@@ -5,20 +5,28 @@
 
 - Traceroute to a host:
 
-`traceroute {{host}}`
+`traceroute {{example.com}}`
 
 - Disable IP address and host name mapping:
 
-`traceroute -n {{host}}`
+`traceroute -n {{example.com}}`
 
 - Specify wait time for response:
 
-`traceroute -w {{0.5}} {{host}}`
+`traceroute --wait={{0.5}} {{example.com}}`
```

Comment:
Is this wait time in seconds ? I think we should update the unit as well

Reply:
>   -w MAX,HERE,NEAR  --wait=MAX,HERE,NEAR
                              Wait for a probe no more than HERE (default 3)
                              times longer than a response from the same hop,
                              or no more than NEAR (default 10) times than some
                              next hop, or MAX (default 5.0) seconds (float
                              point values allowed too)

Yes, is in seconds.

Reply:
I have now written the unit to it.